### DISCUSSION TITLE
Is there an extention that can save logs as they generate? Or something?

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#3826] [Q&A]
Is there an extension that can save logs as they generate? So that if power goes out etc. You can not loose a chat? 

Is there no way to currently restore a current session etc? No automatic log etc? I don't think there is but this is a feature I would love to see.

**Comment by missionfloyd** [comment]
Chats are automatically saved to the logs folder and are loaded when the corresponding character is selected.
